bin/strata-client/src/rpc_server.rs

```diff
@@ -110,6 +112,23 @@ impl StrataRpcImpl {
             .map_err(Error::Db)
             .await
     }
+
+    fn fetch_bridge_state_from_asm(&self) -> Result<BridgeV1State, Error> {
```

Comment:
We need to have these in the new binary as well. Would it make more sense to have these in the new binary since we will be getting rid of this old one?